metashare/api/serializers.py

```diff
@@ -60,6 +61,9 @@ class Meta:
             "old_slugs",
         )
 
+    def get_repo_url(self, obj) -> Optional[str]:
+        return f"https://github.com/{obj.repo_owner}/{obj.repo_name}"
```

Comment:
I don't love this. We _could_ store it on the model and populate it lazily (like we're doing for `repo_id`).